galaxy_ng/app/api/v3/viewsets/collection.py

```diff
@@ -281,6 +281,18 @@ def move_content(self, request, *args, **kwargs):
         add_task = self._add_content(collection_version, dest_repo)
         remove_task = self._remove_content(collection_version, src_repo)
 
+        if (
+            settings.GALAXY_DEPLOYMENT_MODE == DeploymentMode.INSIGHTS.value and
```

Comment:
Could this use galaxy_ng.app.api.permissions.RestrictOnCloudDeployments  or it's friends?